frontend/src/metabase/admin/permissions/constants/data-permissions.js

```diff
@@ -0,0 +1,29 @@
+import { t } from "ttag";
+import { color } from "metabase/lib/colors";
+
+export const DATA_PERMISSION_OPTIONS = {
+  all: {
+    label: t`Allowed`,
+    value: "all",
+    icon: "check",
+    iconColor: color("success"),
```

Comment:
not totally sure if this will work with whitelabeling clobbering what color these strings map to